libs/tailwindcss-config/src/theme.js

```diff
@@ -10,6 +10,33 @@ module.exports = {
   colors: {
     transparent: 'transparent',
     current: 'currentColor',
+    bullish: '#26FF8A',
+    bearish: '#ED1515',
+    vega: {
+      yellow: '#EDFF22',
+      pink: '#FF2D5E',
+      green: '#00F780',
+    },
+    red: {
+      DEFAULT: '#ED1515',
+      transparent: 'rgba(255, 38, 65, 0.3)',
+      vega: '#FF261A',
+    },
+    green: {
+      DEFAULT: '#26FF8A',
+      transparent: 'rgba(38, 255, 138, 0.3)',
+      dark: '#246340', // Same as GREEN_TRANSPARENT given a background of #1f1f1f
```

Comment:
suggestion (non blocking): Lets just remove this comment, dont think its valid anymore
```suggestion
      dark: '#246340',
```